test/skadi_service.cc

```diff
@@ -174,8 +175,11 @@ void create_tile() {
 
   // a place to store it
   auto tile_dir = cfg.get<std::string>("additional_data.elevation");
-  if (!filesystem::is_directory(tile_dir) && !filesystem::create_directories(tile_dir))
+  if (std::filesystem::exists(tile_dir) && !std::filesystem::is_directory(tile_dir)) {
     throw std::runtime_error("Couldnt make directory to store elevation");
+  } else if (!std::filesystem::exists(tile_dir) && !std::filesystem::create_directories(tile_dir)) {
+    throw std::runtime_error("Couldnt make directory to store elevation");
+  }
```

Comment:
this fails OSX CI for some reason @cvvergara . I don't know why. can you try reverting?